Mathlib/Algebra/Polynomial/SumIteratedDerivative.lean

```diff
@@ -0,0 +1,263 @@
+/-
+Copyright (c) 2022 Yuyang Zhao. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yuyang Zhao
+-/
+import Mathlib.Algebra.Polynomial.AlgebraMap
+import Mathlib.Algebra.Polynomial.BigOperators
+import Mathlib.Algebra.Polynomial.Degree.Lemmas
+import Mathlib.Algebra.Polynomial.Derivative
+import Mathlib.Algebra.Polynomial.RingDivision
+
+/-!
+# Sum of iterated derivatives
```

Comment:
Could you add a short summary of the main definition(s) and theorem(s), and what these are useful for?

Reply:
> and what these are useful for?

This would be good to have in the module doc, before the list of definitions.